lib/modules/wdpa/biopama_countries_importer.rb

```diff
@@ -1,18 +1,19 @@
 module Wdpa::BiopamaCountriesImporter
-  BIOPAMA_COUNTRIES_CSV = "#{Rails.root}/lib/data/seeds/biopama_countries_iso_codes.csv"
+  BIOPAMA_COUNTRIES_CSV = "#{Rails.root}/lib/data/seeds/biopama_countries.csv"
   extend self
 
   def import
     ActiveRecord::Base.transaction do
       csv = CSV.read(BIOPAMA_COUNTRIES_CSV)
       csv.shift # remove headers
```

Comment:
I think these two lines can be removed now you are using the `foreach` at line 10

Reply:
Ah, I forgot about those. Fixed now. Thanks, @lucacug !